.github/workflows/pytest.yml

```diff
@@ -125,26 +126,13 @@ jobs:
         env:
           POSTGRES_USER: ckan
           POSTGRES_PASSWORD: ckan
-          POSTGRES_DB: ckan
-          PGDATA: "/data"
-
-      ckan_redis:
-        image: redis:7
-        ports:
-          - 6379:6379
-      ckan_solr:
+      ckan-solr:
         image: ckan/ckan-solr:master
```

Comment:
Its been on my mind that we use 'vanilla' containers for all but solr. Is it at all possible to inject via a shell script or a customer container in this repo the real 'master' def so that you are not fighting the cicd if you want to add additional config to the solr search system?